src/v2/conversions.rs

```diff
@@ -927,6 +952,12 @@ impl TryFrom<AccountInfo> for super::types::AccountInfo {
             None => None,
         };
         let account_address = address.require()?.try_into()?;
+        let mut cds = Vec::with_capacity(cooldowns.len());
+        for cooldown in cooldowns.into_iter() {
+            cds.push(cooldown.try_into()?)
+        }
+        let cooldowns = cds;
+        let available_balance = available_balance.require()?.into();
```

Comment:
I'm assuming here that the balance has to be present, but I'm not sure if this is correct. I'm also not sure how this could interact with old protobuf messages that lack the balance. Presumably they would be considered invalid?

Reply:
Wait... if this can be fully calculated from the other fields, why is it even included in the protobuf message? That would make it redundant, no? We could just add a `available_balance` method to `AccountInfo` to provide the value then.

Reply:
It's technically redundant, but I decided to include it because the means of calculating the available balance has changed, and potentially it could change further in future. Directly providing the available balance helps to future-proof apps. The computation of the available balance is also not entirely trivial, so I think it is useful to provide it (especially for anyone who is directly using the GRPC API).

Reply:
Hmmm... is there a danger of the balance being wrongly set (either unintentionally or intentionally)? I'm wondering if you would you need to verify that the balance is the correct balance, in which case you'd need to calculate it anyway to check.

Reply:
We should trust that the node is reporting the correct data, if it supplies a value.

Reply:
I made it calculate it if it's not present but not sure if the calculation is correct, please check it :)

Also if the correct way to calculate the available balance changes, I guess the calculation here will also be wrong until updated, if no available balance is provided in the protobuf. But I'm guessing that isn't an issue?